201-asev2-ilb-with-web-app/azuredeploy.json

```diff
@@ -0,0 +1,157 @@
+        "description": "Used when deploying an ILB enabled ASE.  Set this to the root domain associated with the ASE.  For example: contoso.com"
+      }
+    },
+    "siteName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the web app that will be created."
+      }
+    },
+    "appServicePlanName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the App Service plan to use for hosting the web app."
+      }
+    },
+    "owner": {
+      "type": "string",
+      "metadata": {
+        "description": "The owner of the resource will be used for tagging."
+      }
+    },
+    "workerPool": {
+      "type": "string",
+      "allowedValues": [
+        "1",
+        "2",
+        "3"
+      ],
+      "defaultValue": "1",
+      "metadata": {
+        "description": "Defines which worker pool's (WP1, WP2 or WP3) resources will be used for the app service plan."
+      }
+    },
+    "numberOfWorkersFromWorkerPool": {
+      "type": "int",
+      "defaultValue": 1,
+      "metadata": {
+        "description": "Defines the number of workers from the worker pool that will be used by the app service plan."
+      }
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2016-09-01",
+      "type": "Microsoft.Web/hostingEnvironments",
+      "name": "[parameters('aseName')]",
+      "kind": "ASEV2",
+      "location": "[parameters('aseLocation')]",
+      "tags": {
+        "displayName": "ASE Environment",
+        "usage": "Hosting PaaS applications",
+        "category": "Environment",
+        "owner": "[parameters('owner')]"
+      },
+      "properties": {
+        "name": "[parameters('aseName')]",
+        "location": "[parameters('aseLocation')]",
+        "ipSslAddressCount": 0,
+        "internalLoadBalancingMode": "[parameters('internalLoadBalancingMode')]",
+        "dnsSuffix": "[parameters('dnsSuffix')]",
+        "virtualNetwork": {
+          "Id": "[parameters('vnetResourceName')]",
+          "Subnet": "[parameters('subnetName')]"
+        }
+      }
+    },
+    {
+      "apiVersion": "2016-09-01",
+      "name": "[parameters('appServicePlanName')]",
+      "type": "Microsoft.Web/serverfarms",
+      "location": "[parameters('aseLocation')]",
+      "tags": {
+        "displayName": "ASE Hosting Plan",
+        "usage": "Hosting Plan within ASE",
+        "category": "Hosting",
+        "owner": "[parameters('owner')]"
+      },
+      "properties": {
+        "name": "[parameters('appServicePlanName')]",
+        "hostingEnvironmentProfile": {
+          "id": "[resourceId('Microsoft.Web/hostingEnvironments/',parameters('aseName'))]"
+        }
+      },
+      "sku": {
+        "name": "[concat('I',parameters('workerPool'))]",
+        "tier": "Isolated",
+        "size": "[concat('I',parameters('workerPool'))]",
+        "family": "I",
+        "capacity": "[parameters('numberOfWorkersFromWorkerPool')]"
+      },
+      "dependsOn": [
```

Comment:
move this above properties

Reply:
What do you mean with move? For as far as I know this property should be there.

Reply:
change the order of properties so that this property comes before the properties object of the resource.

Reply:
done